src/Bundle.ts

```diff
@@ -12,68 +12,349 @@ import transformBundle from './utils/transformBundle';
 import collapseSourcemaps from './utils/collapseSourcemaps';
 import callIfFunction from './utils/callIfFunction';
 import error from './utils/error';
-import {
-	dirname,
-	isRelative,
-	isAbsolute,
-	normalize,
-	relative,
-	resolve
-} from './utils/path';
+import { normalize, resolve } from './utils/path';
 import { OutputOptions } from './rollup/index';
 import { RawSourceMap } from 'source-map';
 import Graph from './Graph';
 import ExternalModule from './ExternalModule';
-import ExternalVariable from './ast/variables/ExternalVariable';
 import ExportDefaultVariable from './ast/variables/ExportDefaultVariable';
+import Variable from './ast/variables/Variable';
+import NamespaceVariable from './ast/variables/NamespaceVariable';
+import ExternalVariable from './ast/variables/ExternalVariable';
+import { makeLegal } from './utils/identifierHelpers';
 import { DynamicImportMechanism } from './ast/nodes/Import';
 
+export type BundleDependencies = {
+	id: string;
+	name: string;
+	isBundle: boolean;
+	reexports?: ReexportSpecifier[];
+	imports?: ImportSpecifier[];
+}[];
+
+export type BundleExports = {
```

Comment:
This is because the shape is an array shape, which can't be represented by an interface (interfaces are objects types only).

Reply:
Ah I see. I guess I missed the array brackets :). Still, it could be
```ts
interface ChunkDependency = {
	id: string;
	name: string;
	isChunk: boolean;
	reexports?: ReexportSpecifier[];
	imports?: ImportSpecifier[];
}
export type ChunkDependencies = ChunkDependency[]
```
or maybe not assign a special type to the array at all.